<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QMainWindow" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="xPosLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>X position:</string>
    </property>
   </widget>
   <widget class="QLabel" name="VarValLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="VarNumLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Variable #:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yPosLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Y position:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="xText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="yText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="variableText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="valueText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
